recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - blosc
```

Comment:
Hmm this doesn't seem right... blosc should not be needed in build.

Reply:
If blosc is not inside the build environment, during conda build, PyTables will not be able to find it when it runs its setup.py. When PyTables can't find Blosc it goes through the logic of working around it including using the internally copied libraries here: https://github.com/PyTables/PyTables/tree/master/c-blosc/internal-complibs. These libraries may not match the conda environment and cause segmentation faults.

@avalentino [mentioned the fix](https://github.com/PyTables/PyTables/issues/686#issuecomment-385045844) in [PyTables Issue 686](https://github.com/PyTables/PyTables/issues/686) originally but it wasn't fixed completely since it was blosc was missing from the build section.

Specifically it was mentioned:
> In short IMO the problem could be fixed by adding blosc to the requirements (build and run) of the pytables-feedstock recipe.

The build requirement seemed to be erroneously omitted.

Reply:
Hmm that seems like a bug in PyTables then.  Can you add a comment in the meta.yaml that captures this?

Reply:
@avalentino from his comment in [PyTables Issue 686 ](https://github.com/PyTables/PyTables/issues/686) seems to disagree. 

> I'm closing this issue because it seems to be a packaging issue more than a PyTables bug 

I am not a Pytables developer so I cannot speak for their intended behavior. I believe they understand that these segmentation faults are due to the missing blosc package fallback workaround causing mismatching symbols in the environment. I can still add a comment but I'm not in a PyTables maintainer position to know what to label the issue as.

Reply:
Just add the above as a comment about the line in the build section. Thanks!

Reply:
I've added a comment. Let me know if there's any additional clarification or simplification that you would like.